source/dlangbot/warnings.d

```diff
@@ -14,6 +14,27 @@ struct UserMessage
 }
 
 
+void checkEnhancementChangelog(in ref PullRequest pr, ref UserMessage[] msgs,
```

Comment:
Don't use `in ref` please

Reply:
I'm using the same signature as the method below.